l10n_it_delivery_note/tests/delivery_note_common.py

```diff
@@ -4,8 +4,15 @@
 
 
 class StockDeliveryNoteCommon(TransactionCase):
-    def create_partner(self, name, **kwargs):
-        return self.env["res.partner"].create({"name": name, **kwargs})
+    def create_partner_commercial(self, name, **kwargs):
```

Comment:
potresti per favore rinominarlo in `create_commercial_partner`? In modo che sia coerente con il field `commercial_partner_id`. Grazie.